### ISSUE TITLE
Excuse me, ask a few questions

### ISSUE THREAD

**Issue by tlerbao** [HighTechLowIQ/ModdingDiablo2Resurrected#2]
1. What software or method is used to edit the .sprite file.
2. Use the -direct -txt method, every time you modify the .txt save file, whether you want to completely exit the game, restart the game before going home and reconfiguration.

Many thanks

**Comment by HighTechLowIQ** [comment]
> What software or method is used to edit the .sprite file.

Check out [SpriteEdit](https://github.com/eezstreet/D2RModding-SpriteEdit/releases/tag/v1.1.0) - that's the only tool I'm aware of so far.

> Use the -direct -txt method, every time you modify the .txt save file, whether you want to completely exit the game, restart the game before going home and reconfiguration.

I'm not sure what you mean by this? Could you rephrase it?

**Comment by tlerbao** [comment]
@HighTechLowIQ Thank you, so how do I modify the .srt file?

**Comment by HighTechLowIQ** [comment]
> Thank you, so how do I modify the .srt file?

You can just edit .srt files with any text editor. They're general subtitle files.

**Comment by tlerbao** [comment]
Many thanks

**Comment by tlerbao** [comment]
Many thanks
